cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml

```diff
@@ -124,26 +124,6 @@ data:
       # When this is enabled (enabled = true), persistent volume metrics including those in the kube-system namespace will be collected
       enabled = true
 
-  alertable-metrics-configuration-settings: |-
```

Comment:
Any chance you can leave this section headers in but commented out with a note that these are now being set through `Microsoft.AlertsManagement/prometheusRuleGroups` alerts in bicep, external to the cluster.

Not a big deal if you don't; but might help connect the dots between the two approaches

Reply:
thanks for this comment @ckittel, while at the very beginning we followed the "commenting out approach", we had decided moving forward deleting this configuration from the `ConfigMap` ^. 

The rationale behind this is that ContainerInsights alter "legacy" mechanism is being [retired (metric alerts) without reaching GA](https://learn.microsoft.com/en-gb/azure/azure-monitor/containers/kubernetes-metric-alerts?tabs=portal#legacy-container-insights-metric-alerts-preview).

If my understanding is wrong, please just let me know and I will be happy to revert this commit 2023af78bbecf930383b2483d2ce8757cd4accdf

Please note that for ConfigMap vs DCR confirmation log confirmation we are keeping the lines commented out as discussed (different PR).

Reply:
Agreed with removal!  Closing suggestion as "do not do :)"